### DISCUSSION TITLE
Weird dialogs are shown when pdf is loaded

### DISCUSSION THREAD

**Discussion by gotteman** [stephanrauh/ngx-extended-pdf-viewer#2111] [Q&A]
Hi,

Does anybody know how to get rid of the below dialogs when the pdf is loaded? They are hidden when I click the document but I don't want them to be visible at all. 

Thanks in advance!

![image](https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/1047129/64e45111-e2ea-45ee-8e97-347fa75b61b2)

**Comment by stephanrauh** [comment]
I think it's the other way round. You don't have to hide the dialogs - you have to find out why they're visible. By default, they're not. I suspect you've got a CSS rule that accidentally shows `<dialog>` elements. The browser hides dialog by default, but many CSS frameworks do a "reset", and that might by the problem.
